app/full_text_search.py

```diff
@@ -99,22 +99,25 @@ def create_romanized_query_txt(query_txt):
     return enclose_in_parenthesis(romanized_query)
 
 
-def is_query_match(query_txt, values_romanized_unidecode,
-                   values_romanized_jp_words):
+def search_terms(query_txt, values_romanized_unidecode,
```

Comment:
ah, sorry, I meant reaname the "words" variable to search_terms. The name of this function was ok!